### PR TITLE
Documentation for Pythagorean Tree and Forest

### PR DESCRIPTION
**Pull request by ajdapretnar**
I also threw out ModelMap Projection Rank since it's no longer in docs.

### CODE REVIEW

File: doc/visual-programming/source/widgets/visualize/pythagoreanforest.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Pythagorean forest for visualising random forests.
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Random Forest**
+
+Classification / regression tree models as random forest.
```
Comment (kernc) [inline doc/visual-programming/source/widgets/visualize/pythagoreanforest.rst]: This should be indented.

File: doc/visual-programming/source/widgets/visualize/pythagoreanforest.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+**Outputs**:
+
+-  **Tree**
+
+A selected classification / regression tree model.
```
Comment (kernc) [inline doc/visual-programming/source/widgets/visualize/pythagoreanforest.rst]: Indented.

Comment (ajdapretnar) [inline doc/visual-programming/source/widgets/visualize/pythagoreanforest.rst]: Nope.

File: doc/visual-programming/source/widgets/visualize/pythagoreanforest.rst (hunk 1 above)
Comment (ajdapretnar) [inline doc/visual-programming/source/widgets/visualize/pythagoreanforest.rst]: Nope.

File: doc/visual-programming/source/widgets/visualize/pythagoreantree.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
+Box Plot
```
Comment (astaric) [inline doc/visual-programming/source/widgets/visualize/pythagoreantree.rst]: Box plot?

**Comment by astaric** [comment]
Please include the new pages in the corresponding index.rst. (https://github.com/ajdapretnar/orange3/blob/pythagorean-docs/doc/visual-programming/source/index.rst)

File: doc/visual-programming/source/widgets/visualize/modelmapprojectionrank.rst [hunk 4]
```diff
@@ -1,71 +0,0 @@
-ModelMap Projection Rank
-========================
-
-.. figure:: icons/modelmap-projection.png
```
Comment (astaric) [inline doc/visual-programming/source/widgets/visualize/modelmapprojectionrank.rst]: Images used only by this file should also be removed. If possible, removal of file and corresponding images should be moved to a separate commit.

**Comment by ajdapretnar** [comment]
All fixed.

File: doc/visual-programming/source/widgets/visualize/pythagoreanforest.rst (hunk 1 above)
Comment (kernc) [inline doc/visual-programming/source/widgets/visualize/pythagoreanforest.rst]: It should be indented. It's easy to see the fault if you process it into HTML, or just [preview the result](https://github.com/biolab/orange3/pull/1511/files?short_path=d23dea2#diff-d23dea23dcda62bd206bfa2dd75c66bf) (locate this file once the page loads) here on Github. The description falls out of the list (and a new list is begun if more than one signal). This is clearly unprofessional, semantically incorrect, and it looks the weirder, depending upon the stylesheet.

This is how it should look like: https://github.com/biolab/orange3/blob/3ba5ab2e032440d516dbb916caf50a47194d94de/doc/widgets/classify/classificationtree.md

This is how it looks now: https://github.com/biolab/orange3/blob/master/doc/visual-programming/source/widgets/classify/classificationtree.rst

Let me know if you can spot the difference.

Comment (ajdapretnar) [inline doc/visual-programming/source/widgets/visualize/pythagoreanforest.rst]: It's done documentation-wide so I'm not changing it in this PR. Could be fixed in a subsequent PR that addresses indentation only.

**Comment by BlazZupan** [comment]
Please include `widgets/visualize/pythagoreantree` and `widgets/visualize/pythagoreanforest` in `doc/visual-programming/source/index.rst`.

**Comment by ajdapretnar** [comment]
Strange, it must have not made it in my previous commit. Now it really is fixed.
